mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -773,14 +731,6 @@ impl MmCtxBuilder {
 
         if let Some(conf) = self.conf {
             ctx.conf = conf;
-
-            let event_stream_configuration = &ctx.conf["event_stream_configuration"];
-            if !event_stream_configuration.is_null() {
-                let event_stream_configuration: EventStreamConfiguration =
-                    json::from_value(event_stream_configuration.clone())
-                        .expect("Invalid json value in 'event_stream_configuration'.");
-                ctx.event_stream_configuration = Some(event_stream_configuration);
-            }
```

Comment:
aha this removes the ability for a user to specifically disable streaming. we could have an mm2 config param to set this up.

what's a use case for such a config though? all of the event streamers data are already accessible via normal rpc. one wouldn't compromise anything by having the streaming accessible. if the streaming endpoint is public for some reason so is the rpc which poses the same issues.

Reply:
haha, this actually made me notice we never auth the SSE connection lol [fc133d7](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/fc133d775b2ef19d899ad8517af48366cd8b6968)
will try to cover it today

Reply:
> what's a use case for such a config though? all of the event streamers data are already accessible via normal rpc. one wouldn't compromise anything by having the streaming accessible. if the streaming endpoint is public for some reason so is the rpc which poses the same issues.

They are 2 different communication layers, each should be configurable (including CORS policies) individually. An example to this use case would be this: We have playground nodes that runs publicly exposed RPCs and we don't need to expose SSE there. It's an edge use case for sure, but when you think as a consumer, wouldn't that be good to see that you can permanently switch off things that you don't need?

Reply:
to be reintroduced and contains `worker_path` & `rpccors` configs.
SSE endpoint disable if this is `null`.
SSE endpoint needs auth.

Reply:
covered here [bb90b65](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/bb90b651ed853c9498aecde2ab25dfc8ac8171d8)
(but no auth yet, we might as well skip it since enablers already auth)